File: data/src/main/java/soup/movie/data/model/Trailer.java
package soup.movie.data.model;

import android.support.annotation.Keep;

import com.google.gson.annotations.SerializedName;

@Keep
public class Trailer {

    @SerializedName("author")
    private String author;

    @SerializedName("title")
    private String title;

    @SerializedName("youtube_id")
    private String id;

    @SerializedName("thumbnails")
    private Thumbnails thumbnails;

    public Trailer() {
    }

    public String getAuthor() {
        return author;
    }

    public String getTitle() {
        return title;
    }

    public String getId() {
        return id;
    }

    public Thumbnails getThumbnails() {
        return thumbnails;
    }

    @Override
    public String toString() {
        return "Trailer{" +
                "author='" + author + '\'' +
                ", title='" + title + '\'' +
                ", id='" + id + '\'' +
                ", thumbnails=" + thumbnails +
                '}';
    }
}

File: data/src/main/java/soup/movie/data/model/Trailer.kt
package soup.movie.data.model

import androidx.annotation.Keep
import androidx.core.text.HtmlCompat
import com.google.gson.annotations.SerializedName

@Keep
data class Trailer(
    val youtubeId: String,
    @SerializedName("title")
    private val _title: String,
    val author: String,
    val thumbnailUrl: String
) {
    val title: String
        get() = HtmlCompat.fromHtml(_title, HtmlCompat.FROM_HTML_MODE_LEGACY).toString()
}
